go.mod

```diff
@@ -3,15 +3,15 @@ module github.com/talos-systems/capi-utils
 go 1.16
 
 require (
-	github.com/spf13/cobra v1.1.3
-	github.com/spf13/viper v1.7.0
+	github.com/spf13/cobra v1.2.1
+	github.com/spf13/viper v1.8.1
 	github.com/talos-systems/go-debug v0.2.1
 	github.com/talos-systems/go-retry v0.3.1
 	github.com/talos-systems/talos/pkg/machinery v0.11.5
```

Comment:
we could also bump Talos machinery? or it doesn't matter now?

Reply:
surely doesn't hurt to bump it :+1: